adserver/tasks.py

```diff
@@ -581,6 +590,57 @@ def notify_of_completed_flights():
             flight.live = False
             flight.save()
 
+            completed_flights_by_advertiser[flight.campaign.advertiser.slug].append(
+                flight
+            )
+
+    # Send notification to advertiser - one email even if multiple flights finished
+    if settings.FRONT_TOKEN and settings.FRONT_AUTHOR and settings.FRONT_CHANNEL:
```

Comment:
Seems like we should probably have this logic in one place, and then `settings.FRONT_ENABLED` as an alias?